docker-compose.yml

```diff
@@ -16,6 +16,8 @@ services:
     command:  mitmdump -p 8082 --set confdir=/system-tests/utils/proxy/.mitmproxy -s /system-tests/utils/proxy/forwarder.py
     ports:
       - "8082:8082"
+    extra_hosts:
```

Comment:
Why do you need this ?

Reply:
It makes it easier to attach the debugger, but I can remove it.